packages/draw/cypress/e2e/visualtesting.cy.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+describe('mindgraph visual tests', () => {
```

Comment:
Can we add https://www.npmjs.com/package/eslint-plugin-cypress and include these js files in our lint/format checks?

Reply:
Yes! Good callout! added in https://github.com/jollyjerr/mindgraph/pull/11/commits/b17c9688d2109b92b65ba1011de96d7e8d3c52a0